lib/tasks_private/openstack.rake

```diff
@@ -34,4 +34,65 @@ namespace :vcr do
     Rake::Task['vcr:spec:run'].invoke
     Rake::Task['vcr:credentials:obfuscate'].invoke
   end
+
+  task :rerecord_v2 => [] do
+    require 'fog/openstack'
+
+    host = YAML.load_file("config/secrets.yml").dig("test", "openstack", "hostname")
+    port = YAML.load_file("config/secrets.yml").dig("test", "openstack", "port")
+    username = YAML.load_file("config/secrets.yml").dig("test", "openstack", "userid")
+    password = YAML.load_file("config/secrets.yml").dig("test", "openstack", "password")
+
+    `rm spec/vcr_cassettes/manageiq/providers/openstack/cloud_manager/refresher.yml`
+    `rm spec/vcr_cassettes/manageiq/providers/openstack/network_manager/refresher.yml`
```

Comment:
Can you reuse the https://github.com/ManageIQ/manageiq-providers-openstack/pull/814/files#diff-bafe67d15a26691f184fadd47493d35e585f1f5dc3f283c7372b091b7c6ef5feR32 rake task?

Reply:
If you're referring to https://github.com/ManageIQ/manageiq-providers-openstack/blob/master/lib/tasks_private/openstack.rake#L32 in the link above, that would also delete all the other refresh VCRs i.e. `refresher_rhos*.yml`. The new rake task doesn't regenerate those type of VCRs and I'm not sure if there is a way to invoke the delete VCR rake task in a way that would only delete `refresher.yml` files

Reply:
Yeah eventually (sooner rather than later maybe) we'll need to drop those since the first time we make a change to the openstack refresh to collect something new they will fail.